src/llmcompressor/transformers/finetune/data/data_args.py

```diff
@@ -154,13 +154,13 @@ class DataTrainingArguments(CustomDataTrainingArguments):
         },
     )
     min_tokens_per_module: Optional[float] = field(
-        default=0.2,
+        default=None,
```

Comment:
Why are we changing the default here?

Reply:
This is calculated during run time now and defaults to 1/num_experts; since this would be dependent on the model